ospec/README.md

```diff
@@ -10,7 +10,7 @@ Noiseless testing framework
 - ~360 LOC including the CLI runner
 - terser and faster test code than with mocha, jasmine or tape
 - test code reads like bullet points
-- assertion code follows [SVO](https://en.wikipedia.org/wiki/Subject–verb–object) structure in present tense for terseness and readability
+- assertion code follows [SVO](https://en.wikipedia.org/wiki/Subject%E2%80%93verb%E2%80%93object) structure in present tense for terseness and readability
```

Comment:
Odd that this is an issue, but I'll accept it anyways. Also, file a corresponding PR against https://github.com/MithrilJS/ospec - I'm in the process of moving all future development there, and starting with its next update, npm will point there, too.